packages/cli-platform-apple/src/commands/runCommand/getSDKNameFromPlatform.ts

```diff
@@ -0,0 +1,16 @@
+function getSDKNamefromPlatform(platform: string) {
+  switch (platform) {
+    case 'ios':
+      return ['iphonesimulator', 'iphoneos'];
+    case 'tvos':
+      return ['appletvsimulator', 'appletvos'];
+    case 'visionos':
+      return ['xrsimulator', 'xros'];
+    case 'macos':
+      return ['macosx'];
+    default:
+      return ['iphonesimulator'];
+  }
+}
```

Comment:
nit: maybe we could create one function that will return combined output for `getPlatformReadableName` and `getSDKNamefromPlatform` e.g.

```
case: 'ios':
    return {
         name: 'iOS',
         sdkNames: ['iphonesimulator', 'iphoneos']
   }
```